xbmc/pvr/timers/PVRTimerInfoTag.cpp

```diff
@@ -481,26 +676,45 @@ CPVRTimerInfoTagPtr CPVRTimerInfoTag::CreateFromEpg(const CEpgInfoTagPtr &tag)
   /* set the timer data */
   CDateTime newStart = tag->StartAsUTC();
   CDateTime newEnd = tag->EndAsUTC();
-  newTag->m_iClientIndex      = -1;
-  newTag->m_strTitle          = tag->Title().empty() ? channel->ChannelName() : tag->Title();
-  newTag->m_iChannelNumber    = channel->ChannelNumber();
-  newTag->m_iClientChannelUid = channel->UniqueID();
-  newTag->m_iClientId         = channel->ClientID();
-  newTag->m_bIsRadio          = channel->IsRadio();
-  newTag->m_iGenreType        = tag->GenreType();
-  newTag->m_iGenreSubType     = tag->GenreSubType();
-  newTag->m_channel           = channel;
+  newTag->m_iClientIndex       = -1;
+  newTag->m_iParentClientIndex = 0;
```

Comment:
if you add new stuff, then please don't use magic numbers for them, but #define them somewhere